app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+const { getTopics } = require("./controllers/topics.controller.js");
+
+app.get("/api/topics", getTopics);
+
+app.use((request, response, next) => {
```

Comment:
by doing this you are accidentally diagnosing all your unknown 500 error as this. You need an app.all("*", (req, res) => { } )